superset-frontend/src/explore/components/controls/SelectControl.jsx

```diff
@@ -18,22 +18,21 @@
  */
 import React from 'react';
 import PropTypes from 'prop-types';
-import { t, css } from '@superset-ui/core';
-import { Select, CreatableSelect, OnPasteSelect } from 'src/components/Select';
+import { css } from '@superset-ui/core';
+import { Select } from 'src/components';
 import ControlHeader from 'src/explore/components/ControlHeader';
 
 const propTypes = {
+  ariaLabel: PropTypes.string,
```

Comment:
All props here are cleaned and then mapped with the Select ones. This is to minimize impact as the SelectControl component is used dynamically throughout Superset and Superset-ui